.eslintrc.js

```diff
@@ -73,7 +73,8 @@ module.exports = {
       { blankLine: 'always', prev: '*', next: 'break' },
       { blankLine: 'always', prev: '*', next: 'continue' },
       { blankLine: 'always', prev: '*', next: 'throw' }
-    ]
+    ],
+    'prettier/prettier': ['error', { endOfLine: 'auto' }]

```

Comment:
Isn't it already added as part of the previous bug fix?

Reply:
Yup sorry i forget to take the latest pull

Reply:
Np, please update.